src/components/ItemList/__tests__/ItemRow-test.tsx

```diff
@@ -15,6 +16,12 @@ describe('components/ItemList/ItemRow', () => {
 
     const getWrapper = (props = {}): ShallowWrapper => shallow(<ItemRow {...defaults} {...props} />);
 
+    let reduxSpy: jest.SpyInstance;
+
+    beforeEach(() => {
+        reduxSpy = jest.spyOn(ReactRedux, 'useSelector').mockImplementation(() => true);
```

Comment:
Is this `mockImplementation` needed?

Reply:
Yeah it is needed since we use `shallow` above instead of `mount`.